cst438_registration/src/test/java/com/cst438/SystemTestRegistrationService.java

```diff
@@ -0,0 +1,242 @@
+package com.cst438;
+
```

Comment:
These tests are very well-written and comprehensible. The comments are simple, yet efficient in explaining the corresponding code. Overall, these tests look good, and they pass on my end.